bin/util.py

```diff
@@ -810,6 +810,27 @@ def write_yaml(data, path, allow_yamllib=False):
         )
 
 
+# Parse validation mode
+def parse_validation(mode):
```

Comment:
The plan is to drop `validation: custom/default/custom interactive` from `problem.yaml` and infer this via the presence of `output_validator/`, and move `interactive` and `multipass` to a new field `type: [list of, interactive, pass-fail, scoring, multipass]`.

Reply:
yes, we can do that later, i guess?